roles/openshift_health_checker/openshift_checks/efk_check.py

```diff
@@ -0,0 +1,88 @@
+# pylint: disable=missing-docstring
+"""
+Util functions for performiong checks on an Elasticsearch, Fluentd, and Kibana stack
+"""
+
+import json
+import os
+
+from openshift_checks import OpenShiftCheckException, get_var
+
+
+class EFKCheck(object):
+
+    @staticmethod
+    def is_first_master(task_vars):
+        """Run only on first master and only when logging is configured. Returns: bool"""
+        logging_deployed = get_var(task_vars, "openshift_hosted_logging_deploy", default=False)
+        # Note: It would be nice to use membership in oo_first_master group, however for now it
+        # seems best to avoid requiring that setup and just check this is the first master.
+        hostname = get_var(task_vars, "ansible_ssh_host") or [None]
+        masters = get_var(task_vars, "groups", "masters", default=None) or [None]
+        return logging_deployed and masters[0] == hostname
+
+    def get_pods_for_component(self, execute_module, namespace, logging_component, task_vars):
+        """Get all pods for a given component. Returns: list of pods for component, error string"""
+        pod_output = self.exec_oc(execute_module, namespace, "get pods -o json", [], task_vars)
+        try:
+            pods = json.loads(pod_output)
+            if not pods or not pods.get('items'):
+                raise ValueError()
+        except ValueError:
+            # successful run but non-parsing data generally means there were no pods in the namespace
+            return None, 'There are no pods in the {} namespace. Is logging deployed?'.format(namespace)
```

Comment:
This was originally in the context of getting all pods in the namespace and collating.

How about `There are no "{component}" pods in the "{namespace}" project.`

"Is logging deployed" will hopefully be something they ask themselves when all four component checks abort for lack of pods.